### PR TITLE
Fix typos and cleanup

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

File: test/sharness/t0010-basic-commands.sh [hunk 1]
```diff
@@ -67,9 +67,9 @@ test_expect_success "All sub-commands accept help" '
   while read -r cmd
   do
     ${cmd:0:4} help ${cmd:5} >/dev/null ||
-      { echo "$cmd doesnt accept --help"; echo 1 > fail; }
+      { echo "$cmd doesn't accept --help"; echo 1 > fail; }
     echo stuff | $cmd --help >/dev/null ||
-      { echo "$cmd doesnt accept --help when using stdin"; echo 1 > fail; }
```
Comment (Stebalien) [inline test/sharness/t0010-basic-commands.sh:72]: We can't have a single quote here.

File: test/sharness/t0010-basic-commands.sh [hunk 2]
```diff
@@ -82,9 +82,9 @@ test_expect_success "All commands accept --help" '
   while read -r cmd
   do
     $cmd --help >/dev/null ||
-      { echo "$cmd doesnt accept --help"; echo 1 > fail; }
+      { echo "$cmd doesn't accept --help"; echo 1 > fail; }
     echo stuff | $cmd --help >/dev/null ||
-      { echo "$cmd doesnt accept --help when using stdin"; echo 1 > fail; }
```
Comment (Stebalien) [inline test/sharness/t0010-basic-commands.sh:87]: nor here

File: test/sharness_test_coverage_helper.sh [hunk 3]
```diff
@@ -170,7 +170,7 @@ do
 done
 
 # The following will allow us to check that
-# we are properly excuding enough stuff using:
+# we are properly exuding enough stuff using:
```
Comment (Stebalien) [inline test/sharness_test_coverage_helper.sh:173]:
```suggestion
# we are properly excluding enough stuff using:
```

**Comment by rex4539** [comment]
Addressed comments.

**Pull request merged**